owl_msgs/msg/SystemFaultsStatus.msg

```diff
@@ -5,16 +5,17 @@
 # associated message is a bitmask that can represent any subset of
 # these faults occuring at the same time.
 
-# System=1,
```

Comment:
I'm unable to place this comment on line 4, but the full pathname of FaultDetector.h should be given, now that this file has jumped packages.